libs/d3d12core/main.c

```diff
@@ -49,6 +49,8 @@ static pthread_once_t library_once = PTHREAD_ONCE_INIT;
 
 #ifdef _WIN32
 static HMODULE vulkan_module = NULL;
+static HMODULE wineopenxr_module = NULL;
+typedef int (WINAPI *PFN___wineopenxr_GetVulkanExtensions)(uint32_t, uint32_t *, char *);
```

Comment:
Why not just PFN_wineopenxr? Double or triple underscore is reserved IIRC, and I don't see a reason to write uglier code than necessary here.

Reply:
Because the original function is called `__wineopenxr_GetVulkanExtensions` in wineopenxr